crates/linter/src/plugin/strictness/rules/require_strict_types.rs

```diff
@@ -141,7 +143,22 @@ impl Rule for RequireStrictTypesRule {
                 )
                 .with_annotation(Annotation::primary(program.span()))
                 .with_note("The `strict_types` directive enforces strict type checking, which can prevent subtle bugs.")
-                .with_help("Add `declare(strict_types=1);` at the top of your file."),
+                .with_help("Add `declare(strict_types=1);` at the top of your file.")
+                .with_suggestion(program.source, {
```

Comment:
instead of using `with_suggestion`, you should use `context.propose`, example: https://github.com/carthage-software/mago/blob/91146250af4f02e7ad1884b48b19a1098d90ad53/crates/linter/src/plugin/consistency/rules/no_tag_pair_terminator.rs#L55-L57